### ISSUE TITLE
Database Issue

### ISSUE THREAD

**Issue by robertherbaugh** [szybnev/zabbix-compose#1]
I recently deployed your docker file; I'm running into a database error that doesn't seem to have much support on. Can you provide any insight?

The error is that the database is using latin1 instead of UTF8. I've attached a screenshot for reference.
![Screen Shot 2022-02-04 at 15 21 33](https://user-images.githubusercontent.com/63149639/152597953-887ab929-732a-4ee6-a0d6-9c6ded7b3d3c.png)

**Comment by szybnev** [comment]
Yeah, I now about this problem. Thank you for you report. I will fix
this issue as soon as possible.
About error with zabbix agent, it's normal for docker zabbix agent. You
only need to connect it with DNS "zabbix-agent". About this problem you can
find many topics.
